UserApi/UserApi.AcceptanceTests/Steps/AccountSteps.cs

```diff
@@ -117,9 +117,8 @@ private static async Task RemoveGroupFromUserIfExists(AcTestContext testContext)
                 testContext.TestSettings.NewGroups.First().DisplayName, testContext.GraphApiToken);
```

Comment:
Will throw exception if count is 0

Reply:
yeah, that's a temporal dependency. we shouldn't allow this method to be called without having selected a group to remove first. fixed it.